src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -153,4 +155,26 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses a meeting
+     *
+     * @throws ParseException if the give {@code meeting} is invalid.
+     */
+    public static Meeting parseMeeting(LocalDateTime startTime, LocalDateTime endTime, String location)
+            throws ParseException {
+        requireNonNull(startTime);
+        requireNonNull(endTime);
+        requireNonNull(location);
+
+        if (!Meeting.isValidLocation(location)) {
```

Comment:
I feel that these checks are already internalised within Meeting, if you check the Meeting class, it has isValidLocation and isValidStartAndEndTime checks. I could be wrong, let me know whether the extra check is necessary!